backend/src/models/Post.js

```diff
@@ -45,4 +45,5 @@ module.exports = {
     default: () => new Date().toISOString(),
   },
   language: { type: 'string', allow: [null] },
+  checkedBlur: { type: 'boolean', default: false },
```

Comment:
I would rename `checkedBlur` to `blurImage`

Reply:
ok, i changed the name in the whole system